stock_reception_screen_mrp_subcontracting/tests/test_reception_screen.py

```diff
@@ -18,7 +22,106 @@ def setUpClass(cls):
         cls.picking.action_reception_screen_open()
         cls.screen = cls.picking.reception_screen_id
 
+    def _record_components_for_picking(self, picking):
+        for move in picking.move_lines:
+            if not move.is_subcontract:
+                continue
+            context = dict(
+                default_production_id=move.move_orig_ids.production_id.id,
+                default_subcontract_move_id=move.id,
+            )
+            wiz_model = self.env["mrp.product.produce"].with_context(context)
+            wiz = wiz_model.create({})
+            for line in wiz.move_raw_ids.move_line_ids:
+                line.qty_done = line.product_uom_qty
+            wiz.do_produce()
+
     def test_reception_screen_with_subcontracted_products(self):
+        """Receive subcontracted goods."""
```

Comment:
Is it possible to split this? You already added the comments where it would make sense to do so.

Reply:
@sebalix are you going to do it here?

Reply:
TBD in the future IF we keep it for v14